project/Newevaluation.html

```diff
@@ -6,85 +6,118 @@
   <style>
     body{
       background-color: rgb(82, 86, 89);
-      margin-top: 50px;
+      margin-top: 3%;
       margin-left: 0;
     }
 
+
+
+/* ---------------------------------Navigation Bar--------------------------- */
+
     .active{
       border-bottom: 6px solid DodgerBlue;
-      padding:18px;
+      padding: 1%;
     }
 
     nav {
       display: inline;
       background-color: whitesmoke;
-      width: 1000px;
-      height: 100px;
+      width: 100%;
+    //  height: 70%;
     }
 
     .navbar {
-      height: 80px;
+      height: 5rem;
       background-color: whitesmoke;
-      //box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
       border-bottom: 1px solid darkgrey;
-      //box-shadow: 0px 0px 2px 0px lightgrey;
-      //margin-bottom: 1px;
     }
 
     .sv-logo {
-      width: 200px;
-      height: 40px;
-      margin-left: 60px;
-      margin-top: 10px;
+      width: 10%;
```

Comment:
I see you're shooting for a fluid layout. Are you using a grid system to come up with these values? How did you come up with these specific values?